src/client/api/restful/get_output.c

```diff
@@ -115,34 +115,64 @@ static int json_output_deserialize(cJSON *output_obj, utxo_output_t **output) {
   return -1;
 }
 
-res_output_t *get_output_response_new() { return malloc(sizeof(res_output_t)); }
+get_output_t *get_output_new() {
+  get_output_t *output = malloc(sizeof(get_output_t));
+  if (output) {
+    output->is_spent = false;
+    output->ledger_index = 0;
+    output->ml_index_booked = 0;
+    output->ml_index_spent = 0;
+    output->ml_time_booked = 0;
+    output->ml_time_spent = 0;
+    output->output_index = 0;
+    output->output = NULL;
```

Comment:
it's better to use `memset` since values are default to 0.
```suggestion
    memset(output, 0, sizeof(get_output_t);
```